grommet/grommet
# Button fix

What does this PR do?

Changes the logic for the icon in a button to take the color in the theme if one is provided.
Also made a example for storybook using a custom theme and passing in a color to the icon
to provide an example
Where should the reviewer start?

src/js/components/Button/button.js
What testing has been done on this PR?

none
How should this be manually tested?
Any background context you want to provide?
What are the relevant issues?

This is related to issue #2820
Screenshots (if appropriate)
Do the grommet docs need to be updated?

auto
Should this PR be mentioned in the release notes?

no
Is this change backwards compatible or is it a breaking change?

backwards compatible
